racetrack_client/racetrack_client/manifest/validate.py

```diff
@@ -30,17 +32,11 @@ def load_validated_manifest(path: str) -> Manifest:
 
 def validate_manifest(manifest: Manifest):
     """Check whether manifest is valid. Raise exception in case of error"""
-    assert re.match(r"[^@]+@[^@]+\.[^@]+", manifest.owner_email), '"owner_email" is not a valid email'
-
-    with wrap_context('parsing Job version'):
-        SemanticVersion(manifest.version)
-
-    assert ':' in manifest.lang, '"lang" should specify the version in a format "name:version"'
-
-    assert 1 <= manifest.replicas <= 15, 'replicas count out of allowed range'
-
-    assert urlsplit(manifest.git.remote).scheme == 'https', 'git remote URL should be HTTPS'
-
+    serialized_manifest = manifest.dict(exclude_none=True)
+    path = Path(__file__).with_name('schema.json')
+    with path.open('r') as f:
+        schema = json.load(f)
+        validate(serialized_manifest, schema=schema)
```

Comment:
I think we can leave job version validation. Other places in code expect version to be semantic and it may fail if we're going to accept any string.
```suggestion
        validate(serialized_manifest, schema=schema)
        
    with wrap_context('parsing Job version'):
        SemanticVersion(manifest.version)
```

Reply:
I also added this check to the schema.